canonical-webteam/documentation-builder
# Update vanilla-docs-theme to v1.5.3

## Done
Update the embedded vanilla theme to the latest version

## QA
- Run the following commands:
``` bash
python3 -m venv env3 && source env3/bin/activate  # Create encapsulated environment
pip install -e .  # Install the module in editable mode

documentation-builder --source-folder docs  # build the documentation-builder's own documentation
xdg-open build/en/index.html  # Open up the documentation page
```
- Check that the styling looks "interesting" but new
